src/entities/user.entity.ts

```diff
@@ -0,0 +1,13 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity({ name: 'users' })
+export class User {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column({ unique: true })
```

Comment:
I suggest adding index to get more performance:

```suggestion
  @Index()
  @Column({ unique: true })
```

Reply:
Resolved on 8568ec2e0a63a0a8a642d4dabda632c40e4a9f2a.